src/main/java/io/vertx/kafka/admin/KafkaAdminClient.java

```diff
@@ -126,6 +126,16 @@ static KafkaAdminClient create(Vertx vertx, Properties config) {
    */
   Future<Void> deleteTopics(List<String> topicNames);
 
+  /**
+   * Creates a batch of new partitions in the Kafka topic
+   *
+   * @param partitions partitions to create
+   * @param completionHandler handler called on operation completed
+   */
+  @GenIgnore
+  void createPartitions(Map<String, io.vertx.kafka.admin.NewPartitions> partitions, Handler<AsyncResult<Void>> completionHandler);
```

Comment:
you also need the `Future<Void>` version for the master. This one is ok for 3.9 branch only.